Tests/SettingsPageE2ETests.cs

```diff
@@ -0,0 +1,35 @@
+using System.Threading;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Xunit;
+
+namespace Tests
+{
+    [Collection("E2E")]
+    public class SettingsPageE2ETests
+    {
+        [Fact]
+        public void SettingsPageShouldNotContainStartButtonIfScoutIsStarted()
+        {
+            ChromeOptions chromeOptions = new ChromeOptions();
+            chromeOptions.AddArgument("--ignore-certificate-errors");
+            using IWebDriver driver = new ChromeDriver(chromeOptions);
+            driver.Navigate().GoToUrl("https://localhost:5001/Settings");
+            Thread.Sleep(1000);
```

Comment:
Er alle de her sleep, fordi vi venter på at den får connected til url?
man ikke await at den kommer frem?

Reply:
Man kan ikke await, men man kan godt vente indtil et element eksistere. Det kan jeg prøve i stedet for.